CountOnMe/Model/CalculatorManager.swift

```diff
@@ -39,20 +43,33 @@ struct CalculatorManager {
         }
         stringResult += " " + operation + " "
         if operation == "=" {
-            stringResult = calculatorModel.getResult(rawString: stringResult)
+            let unformattedString = calculatorModel.getResult(rawString: stringResult)
+            guard let formattedString = formatString(unformattedString) else { return }
```

Comment:
**suggestion (blocking)**: If for some unknown reason the formatting fails, nothing will happen for the user. You should gracefully handle the case where this issue happen.